conda/recipes/cudf/meta.yaml

```
@@ -41,7 +41,7 @@ requirements:
     - setuptools
     - numba >=0.54
     - dlpack>=0.5,<0.6.0a0
-    - pyarrow =8.0.0 *cuda
+    - pyarrow 8.0.0
```

@shwina I think conda has complained at me before about imprecise package specifications when I removed a `=`. Can we add back the `=`?
```suggestion
    - pyarrow =8.0.0
```

You can see the warning here: https://github.com/conda/conda-build/blob/7be9ba75b6bf0cda07ba2ab48e92b454ad398179/conda_build/utils.py#L1867

It may pop up in CI for the GPU jobs that build the Python package -- those haven't run yet so I'm not sure.

```python
>>> import re
>>> spec_needing_star_re = re.compile(r"([\w\d\.\-\_]+)\s+((?<![><=])[\w\d\.\-\_]+?(?!\*))(\s+[\w\d\.\_]+)?$")
>>> spec_needing_star_re.match("pyarrow 8.0.0")
<re.Match object; span=(0, 13), match='pyarrow 8.0.0'>
>>> spec_needing_star_re.match("pyarrow =8.0.0")  # no match here
>>>
```